frontend/src/scenes/ingestion/Sidebar.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import posthogLogo from 'public/posthog-logo.png'
+import { ingestionLogic } from './ingestionLogic'
+import { useActions, useValues } from 'kea'
+import './IngestionWizard.scss'
+import { InviteMembersButton, SitePopover } from '~/layout/navigation/TopBar/SitePopover'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconArticle, IconQuestionAnswer } from 'lib/components/icons'
+import { HelpType } from '~/types'
+import { LemonDivider } from 'lib/components/LemonDivider'
+
+const HELP_UTM_TAGS = '?utm_medium=in-product-onboarding&utm_campaign=help-button-sidebar'
+
+export function Sidebar(): JSX.Element {
+    const { currentIndex, platform } = useValues(ingestionLogic)
+    const { setVerify, setPlatform } = useActions(ingestionLogic)
+    const { reportIngestionHelpClicked } = useActions(eventUsageLogic)
+
+    return (
+        <div className="IngestionSidebar">
+            <div className="IngestionSidebar__content">
+                <div style={{ paddingLeft: 8 }}>
+                    <img src={posthogLogo} style={{ width: 157, height: 30 }} />
+                </div>
+                <div className="IngestionSidebar__steps">
+                    <b>
+                        <div
+                            className={`mb ${currentIndex === 0 && 'ingestion-current-nav-step'}`}
+                            onClick={() => setPlatform(null)}
+                        >
+                            <span>Get started</span>
+                        </div>
+                        <div
+                            className={`mb ${currentIndex === 1 && 'ingestion-current-nav-step'} ${
+                                !platform && 'nonclickable'
+                            }`}
+                            onClick={() => setVerify(false)}
+                        >
+                            <span>Connect your product</span>
+                        </div>
+                        <div
+                            className={`mb ${currentIndex === 2 && 'ingestion-current-nav-step'} ${
+                                !platform && 'nonclickable'
+                            }`}
+                            onClick={() => {
+                                if (platform) {
+                                    setVerify(true)
+                                }
+                            }}
+                        >
+                            <span>Listen for events</span>
+                        </div>
+                    </b>
+                </div>
+                <div className="sidebar-bottom">
+                    <div className="popover mb">
+                        <SitePopover />
```

Comment:
Most of the actions within the `SitePopover` don't work. For example, try clicking account settings, and the modal just closes.

What actions do we want to enable here?

Also, it seems like the ability to switch teams should be somewhere in here?

Reply:
interesting, the buttons and actions are working for me?

Reply:
Hmm, for me, the URL changes, but then I immediately get redirected back to `/ingestion`. Seem the URL flip in the video below. I think it's because of the redirect logic in `sceneLogic`

https://user-images.githubusercontent.com/4813045/169576896-3aad0eb1-1c9e-403b-8fe8-a72746e8dea1.mov

`

Reply:
@clarkus What do you think about rick's comment here with having the ability to switch between projects during ingestion?

Reply:
It's definitely a complicating factor. Is there a reason a user should be able to switch during ingestion? My concern would be that we don't prominently identify the project being affected by ingestion. If switching is necessary, we might need more prominent identification of the project and clear / obvious ways to switch. We could reuse the utility menu, but it's definitely not optimized for switching in this context.

Reply:
If switching is important, it should probably be isolated to a specific point of the workflow. If you're mid-ingestion and you change projects, the snippets or code you use could change based on that switching. It could easily become unclear what values to use. This feels like something worth chatting about.

Reply:
OK maybe this is something to revisit then if we have extra time this week